opentelemetry.html.md.erb

```diff
@@ -51,9 +54,51 @@ otlp:
       PEM_ENCODED_CERTIFICATE
 ```
 
+### <a id='multiple-exporters'></a> Multiple exporters
+
+The OTel Collector supports defining multiple exporters, each with its own configuration. For example, you can define two OTLP gRPC exporters that egress metrics to different destinations.
+
+```
+otlp:
+  endpoint: 203.0.113.10:4317
+otlp/another:
+  endpoint: 203.0.113.11:4317
+```
+
+<p class="note important">
+<span class="note__title"><strong>Important</strong></span>
+You can specify additional experimental exporters including <code>transform</code>  and <code>filter</code> after enabling the <code>experimental OpenTelemetry Collector features (beta)</code> checkbox.</p>
+
+## <a id='metric-receiver-config'></a> Metric Receiver configuration
```

Comment:
We don't currently allow configuring receivers, we always replace them with the one otlp receiver that forwarder agent sends to. This whole section should probably just cover that users shouldn't list receivers and don't need to include them in their pipelines as they will be included automatically.